src/main/java/com/powsybl/network/conversion/server/NetworkConversionException.java

```diff
@@ -59,6 +60,11 @@ public static NetworkConversionException createEquipmentTypeUnknown(String type)
         return new NetworkConversionException(Type.UNKNOWN_EQUIPMENT_TYPE, "The equipment type : " + type + " is unknown");
     }
 
+    public static NetworkConversionException createVariantIdUnknown(String variantId) {
+        Objects.requireNonNull(variantId);
```

Comment:
useless test : createVariantIdUnknown is always called with non null variantId parameter ...

Reply:
it's a public method maybe better keep it ?

Reply:
restored